Clinical-Genomics/scout
# Replace GQ with VAF in cancer snv variants

Fix #2285 

**How to test**:
1. Locally, go to a cancer SNV variant and note that on master branch the GQ field is -1
2. Switch to this branch and note that the GQ field is replaced by VAF and the numbers are the same as in variantS page as well as in the general report.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
